### ISSUE TITLE
dial in not working properly

### ISSUE THREAD

**Issue by nephilim75** [bigbluebutton/bigbluebutton#16521]
**Describe the bug**
We cannot dial properly. After restarting freeswitch by `systemctl restart freeswitch` it works sometimes and sometimes not. Voicebridge is already enabled to have this feature activated. But it won't work reliable.

**To Reproduce**
1. we have created `/opt/freeswitch/conf/sip_profiles/external/sipgate.xml` and `/opt/freeswitch/conf/dialplan/public/sipgate.xml` and restarted the freeswitch service. Sipgate shows an active device at their end.
2. We have adapted bigbluebutton.properties according to [this page](https://docs.bigbluebutton.org/admin/customize.html#add-a-phone-number-to-the-conference-bridge)
3. Starting a meeting and activate voice bridge by open the mic of the host
4. Make phone call but then nothing happen. No answer, no voice which is prompting to enter the PIN. Just nothing. Silence.

**Expected behavior**
Calling the number we expect the prompt to enter the conference PIN in a reliable manner.

**Actual behavior**
After dialing the umber: No answer, no voice which is prompting to enter the PIN. Just nothing. Silence. But it is not in general. SOmetimes it is working as expected.

**Screenshots**
no screenshots

**Desktop (please complete the following information):**
 - OS: Windows 10, 11 (Chrome, Firefox actual versions)

**Smartphone (please complete the following information):**
 - Device: [e.g. iPhone XR, Pixel 5 - both up to date]

**Additional context**
It looks like restarting freeswitch could "solve" the issue for a short period of time.

**Server and bbb-version**
OS: Ubuntu 20.04
BBB-version: 2.5.11

```
root@bbb2511:~# bbb-conf --status
nginx —————————————————► [✔ - active]
freeswitch ————————————► [✔ - active]
redis-server ——————————► [✔ - active]
bbb-apps-akka —————————► [✔ - active]
bbb-fsesl-akka ————————► [✔ - active]
mongod ————————————————► [✔ - active]
bbb-html5 —————————————► [✔ - active]
bbb-webrtc-sfu ————————► [✔ - active]
kurento-media-server ——► [✔ - active]
kurento-media-server-8888  [✔ - active]
kurento-media-server-8889  [✔ - active]
kurento-media-server-8890  [✔ - active]
bbb-html5-backend@1 ———► [✔ - active]
bbb-html5-backend@2 ———► [✔ - active]
bbb-html5-frontend@1 ——► [✔ - active]
bbb-html5-frontend@2 ——► [✔ - active]
etherpad ——————————————► [✔ - active]
bbb-web ———————————————► [✔ - active]
bbb-pads ——————————————► [✔ - active]
bbb-rap-caption-inbox —► [✔ - active]
bbb-rap-resque-worker —► [✔ - active]
bbb-rap-starter ———————► [✔ - active]
root@bbb2511:~#
```

Looking forward to your assistance.

kind regards
//nephilim

**Comment by antobinary** [comment]
Hi @nephilim75 

Is this something that worked for you correctly with the exact same config changes in 2.5.10 or a previous 2.5 release?
Are you upgrading an existing server or installing a clean one?
Anything interesting near the bottom of `bbb-conf --check`?

**Comment by nephilim75** [comment]
> Is this something that worked for you correctly with the exact same config changes in 2.5.10 or a previous 2.5 release?

Not pretty sure in 2.5.10 as I upgraded as soon as 2.5.11 was pubished. So I didn't tested in production with 2.5.10. But in 2.4 it was working perfectly fine.


> Are you upgrading an existing server or installing a clean one?

It was fresh from scratch.


> Anything interesting near the bottom of `bbb-conf --check?`

No, no potential problems listed there:

```
root@bbb2511:~# bbb-conf --check

BigBlueButton Server 2.5.11 (3193)
                    Kernel version: 5.4.0-137-generic
                      Distribution: Ubuntu 20.04.5 LTS (64-bit)
                            Memory: 32108 MB
                         CPU cores: 8

/etc/bigbluebutton/bbb-web.properties (override for bbb-web)
/usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties (bbb-web)
       bigbluebutton.web.serverURL: https://bbb.***.**
                defaultGuestPolicy: ALWAYS_ACCEPT
                 svgImagesRequired: true
              defaultMeetingLayout: CUSTOM_LAYOUT

/etc/nginx/sites-available/bigbluebutton (nginx)
                       server_name: bbb.***.**
                              port: 80, [::]:80
                              port: 443 ssl

/opt/freeswitch/etc/freeswitch/vars.xml (FreeSWITCH)
                       local_ip_v4: 188.34.***.***
                   external_rtp_ip: 188.34.***.***
                   external_sip_ip: 188.34.***.***

/opt/freeswitch/etc/freeswitch/sip_profiles/external.xml (FreeSWITCH)
                        ext-rtp-ip: $${local_ip_v4}
                        ext-sip-ip: $${local_ip_v4}
                        ws-binding: 188.34.***.***:5066
                       wss-binding: 188.34.***.***:7443

/usr/local/bigbluebutton/core/scripts/bigbluebutton.yml (record and playback)
                     playback_host: bbb.***.**
                 playback_protocol: https
                            ffmpeg: 4.2.7-0ubuntu0.1

/usr/share/bigbluebutton/nginx/sip.nginx (sip.nginx)
                        proxy_pass: 188.34.***.***
                          protocol: http

/usr/local/bigbluebutton/bbb-webrtc-sfu/config/default.yml (Kurento SFU)
/etc/bigbluebutton/bbb-webrtc-sfu/production.yml (Kurento SFU - override)
                        kurento.ip: 188.34.***.***
                       kurento.url: ws://127.0.0.1:8888/kurento
                    kurento.sip_ip: 188.34.***.***
               recordScreenSharing: true
                     recordWebcams: true
                  codec_video_main: VP8
               codec_video_content: VP8

/usr/share/meteor/bundle/programs/server/assets/app/config/settings.yml (HTML5 client)
/etc/bigbluebutton/bbb-html5.yml (HTML5 client config override)
                             build: 2894
                        kurentoUrl: wss://bbb.***.**/bbb-webrtc-sfu
                  enableListenOnly: true
                    sipjsHackViaWs: true

/usr/share/bbb-web/WEB-INF/classes/spring/turn-stun-servers.xml (STUN Server)
                              stun: stun.***.**


# Potential problems described below

root@bbb2511:~#
```

**Comment by hostbbb** [comment]
This appears to be a sip registration issue, and can debugged by looking at the freeswitch console when you make a call.
If you call the number and dont see it hit the dialplan,  sipgate does not have it registered in their switch,

if when you call you see it hit the dialplan,  then the log with show the logic as to why the call did not get matched to join the conference.

suggest working in bigbluetutton-setup to troubleshoot, as most likely not software issue if it works sometimes.

**Comment by nephilim75** [comment]
>  sipgate does not have it registered in their switch,

![grafik](https://user-images.githubusercontent.com/19992865/213865171-e8db3be3-507f-4887-95c4-9e9c4cf102c1.png)
Sipgate dashboard tells me it is registered.

> can debugged by looking at the freeswitch console when you make a call.

Unfortunately I need to ask hot I can check the freeswitch console and how to interact in it? I didn't do that before. May I ask for assistance? I will dig into this afterwards.

**Comment by hostbbb** [comment]
/opt/freeswitch/bin/fs_cli -p $(xmlstarlet sel -t -m ***@***.***="password"]' -v @value /opt/freeswitch/etc/freeswitch/autoload_configs/event_socket.conf.xml)

 

Login as root, and this command above will log you into freeswitch console.

 

When you dial you should see bunch of activity…

 

If you dial and there is no activity means sipgate if not sending the call.

 

If you see activity the reason why not answering is in console.

 

You also can /opt/freeswitch/log and details are in the freeswitch.log

 

Regards,

Stephen

**Comment by nephilim75** [comment]
> ***@***.***="password"

OK, thank you so much for your assistance. One question left: where to find the mentioned credentials. SIP credentials maybe?

**Comment by hostbbb** [comment]
> > ***@***.***="password"
> 
> OK, thank you so much for your assistance. One question left: where to find the mentioned credentials. SIP credentials maybe?

the password is in /opt/freeswitch/etc/freeswitch/autoload_configs/event_socket.conf.xml the command in previous post is one long line that if you enter in shell will get password from file and pass to fs_cli

this is the single line to run, its just wrapping in the post

opt/freeswitch/bin/fs_cli -p $(xmlstarlet sel -t -m ***@***.***="password"]' -v [@value](https://github.com/value) /opt/freeswitch/etc/freeswitch/autoload_configs/event_socket.conf.xml)

**Comment by antobinary** [comment]
Looksl like this was a setup issue. For people who find this in the future -- we recomend https://groups.google.com/g/bigbluebutton-setup as the best spot to receive assitance with setup issues

Thanks @hostbbb for answering here too!

**Comment by nephilim75** [comment]
OK, found the PW in mentioned file. Thought, that I will find the "username" there as well. What is the \*\*\*@\*\*\*.\*\*\* in front of the password?

```
<?xml version="1.0"?>
<configuration name="event_socket.conf" description="Socket Client">
  <settings>
    <param name="nat-map" value="false"/>
    <param name="listen-ip" value="::"/>
    <param name="listen-port" value="8021"/>
    <param name="password" value="***secret***"/>
    <!--<param name="apply-inbound-acl" value="loopback.auto"/>-->
    <!--<param name="stop-on-bind-error" value="true"/>-->
  </settings>
</configuration>
```

**Comment by nephilim75** [comment]
```
root@bbb2511:~# /opt/freeswitch/bin/fs_cli -p $(xmlstarlet sel -t -m ***@***.***="password"]' -v @value /opt/freeswitch/etc/freeswitch/autoload_configs/event_socket.conf.xml)
>
```

Looks like I do something wrong. Once I press Enter a new line appears with `>`. Does it expect any further input?

Anyhow. I now dial. First time I got through, the try after it did not work. In both cases nothing to see on screen... :-(

Then I checked the logfile but being not able to understand it.

Any assistance appreciated. I downloaded the logfile and could share it.
